### PR TITLE
Nickolas-Reverendo-CH New Caption

### LINKED ISSUE #9: Nickolas-Reverendo-CH needs to update caption
**TODO:**
 - [x] update the slide caption
 - [x] verify/update the image
 - [x] update the title

### PR DESCRIPTION
**Pull request by Nickolas-Reverendo-CH**
Updated photo and caption

Closes #9

### CODE REVIEW

File: _posts/0000-01-02-Nickolas-Reverendo-CH.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+If you can dodge a wrench, you can resolve a merge conflict!
```
Comment (beardofedu) [inline _posts/0000-01-02-Nickolas-Reverendo-CH.md]: This is a great caption for the new image!

File: _posts/0000-01-02-Nickolas-Reverendo-CH.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ layout: slide
...
 
-![inflatocat](https://octodex.github.com/images/inflatocat.png)
+![inflatocat](https://octodex.github.com/images/dodgetocat_v2.png)
```
Comment (beardofedu) [inline _posts/0000-01-02-Nickolas-Reverendo-CH.md]: I know you changed the image, but if you want to change it again you can grab images from here: https://octodex.github.com/

File: _posts/0000-01-02-Nickolas-Reverendo-CH.md [hunk 3]
```diff
@@ -3,7 +3,8 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _posts/0000-01-02-Nickolas-Reverendo-CH.md]:
```suggestion
title: "Welcome to Nickolas Reverendo's slide!"
```

File: _posts/0000-01-02-Nickolas-Reverendo-CH.md (hunk 1 above)
Comment (Nickolas-Reverendo-CH) [reply _posts/0000-01-02-Nickolas-Reverendo-CH.md]: It was either that or Bob's Burgers

File: _posts/0000-01-02-Nickolas-Reverendo-CH.md (hunk 2 above)
Comment (Nickolas-Reverendo-CH) [reply _posts/0000-01-02-Nickolas-Reverendo-CH.md]: I will save that link for the future 🔮

**Pull request merged**